lib/microdata/strategy/json_ld.ex

```diff
@@ -30,8 +31,13 @@ defmodule Microdata.Strategy.JSONLD do
     end
   end
 
+  def parse_object(object, parent_context) when is_list(object) do
+    object
+    |> Enum.map(&parse_object(&1, parent_context))
+  end
+
   def parse_object(object, parent_context) do
-    id = object["@id"]
+    id = Map.get(object, "@id")
```

Comment:
Why explicit Map calls?

Reply:
![image](https://user-images.githubusercontent.com/721128/52081119-74073780-2567-11e9-9e0e-3c188574a2c4.png)

Reply:
You're barking up the wrong tree here - the exception you've noted comes from trying to access a keyword list (`[a: 1, b: 2]`) with a non-atom key, which is indeed a valid error. In the case of calling Access based lookup on a map a `"@id"` index is totally valid. Moreover, if you look at any of the callers to this method they're all guaranteed to pass maps to `parse_object/2` so the two implementations are equivalent. 

How you managed to get a keyword list passed into this function is a totally different question, however.

Reply:
Ah—either way, it no longer threw errors. I used `Map.get` because that's how you're reading context below.

Reply:
Ooooo, I take that back—it _was_ being passed a list, insofar as the top-level JSON-LD object was an array. This is probably okay to revert

Reply:
As a style point I generally only use `Map.get` (or `Keyword.get`) in either pipes, or when I want to pass a default value (this is common when pulling from an opts list, like
```
def do_it(arg, other_arg, opts \\ []) do
  verbose = Keyword.get(opts, :verbose, false)
  ...
end
```

Reply:
Right so the error here is that you're passing in a list, which Access infers to be a keyword list (since they're really just lists under the hood). FWIW Map.get would die with a similar exception in this case.

Reply:
Leave this PR with me; I'll get it cleaned up

Reply:
FWIW *I'm* not passing a list—the site presents valid JSON, which the pre-PR code assumed would never be a list ;)